### ISSUE TITLE
Booting failure  OS 1.5.0b2 (also happened with 1.5.0)

### ISSUE THREAD

**Issue by cooptrol** [polyend/TrackerBetaTesting#1420]
This happens almost when I switch it on after some hours of not using it  (see attached video). I bought it 3 weeks ago and it has always happened. After 1 or 2 reboots, it boots ok. Sometimes it boots with the correct graphics but the buttons are unresponsive (also happened with 1.5.0). I have it plugged to the wall wart with the original cable and charger. Since it still works fine after rebooting, I havent made any bigger complaint. I can't return it cos I live in Uruguay and it was delivered here by an intermediary courier. I bought it on ebay from Kraft Music. I guess it's not a software problem, or is it?

https://user-images.githubusercontent.com/100333241/155889475-21df16dc-a187-4492-83a4-67fcee2755ea.mp4

**Comment by Jacknametrouble** [comment]
If you're using the original AC adapter and cable, try using an alternative one, maybe the original AC adapter is causing issues.

**Comment by Jacknametrouble** [comment]
Does this mean, you solved the issue?

**Comment by cooptrol** [comment]
Just tested a different charger and first boot was ok, gonna keep reporting in the next days. Hope this is the solution!

**Comment by cooptrol** [comment]
Seems that was the problem! A shame not to be able to use the original charger, but it works fine with my phone!

**Comment by bufu9k** [comment]
Hi, I’d just like to add that I am able to reproduce this bug very consistently by not shutting the device down properly. If I simply shut off the surge protector the tracker is plugged into, on next boot it will do the wonky graphics. 

Sometimes it takes two or three tries afterwards but I’m always able to get it back to a proper boot and into a stable state. 

This happened the very first time I powered up the device and I was FLOORED because I thought I got a defective unit. Nope, just something weird. Can’t tell if it’s hardware or software. 

v1.5.0. I’ve also tried the original power adapter, an official apple adapter and I tried via directly plugging into a wall socket or through a surge protector and the same result happens.
